app/assets/stylesheets/new_board/_story.scss

```diff
@@ -358,6 +358,19 @@
             .edit-description-text {
               resize: vertical;
               min-height: 100px;
+
+              &__suggestions {
+                position: absolute !important;
```

Comment:
Do we need to use `!important` to _all_ of them? It seems like something's wrong about it.

Reply:
You was right. We need of `!important` just in `background` for replace the default style from `react-mention`